odoo80/Dockerfile

```diff
@@ -1,51 +1,5 @@
-FROM vauxoo/docker-ubuntu-base
+FROM vauxoo/docker-ubuntu-base:latest
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN wget -qO- https://deb.nodesource.com/setup | sudo bash - \
-    && apt-get upgrade -q \
-    && apt-get install --allow-unauthenticated -q \
-    antiword \
-    bash-completion \
-    libfreetype6-dev \
-    libgeoip-dev \
-    libjpeg-dev \
-    libldap2-dev \
-    libqrencode-dev \
-    libxml2-dev \
-    libxslt1-dev  \
-    libyaml-dev \
-    M2Crypto \
-    nodejs \
-    phantomjs \
-    poppler-utils \
-    python-imaging \
-    python-libxml2 \
-    python-lxml \
-    python-pandas \
-    python-qrcode \
-    ttf-dejavu \
-    xmlstarlet \
-    xsltproc \
-    zlib1g-dev \
-    && npm -g install npm@latest \
-    && npm install -g less \
-    && npm install -g less-plugin-clean-css \
-    && pip install --upgrade pyyaml pillow GeoIP SOAPpy pyopenssl suds xmltodict recaptcha-client egenix-mx-base git+https://github.com/Vauxoo/panama-dv.git \
-    && cd /tmp && git clone --depth=1 https://github.com/thewtex/sphinx-contrib.git \
-    && cd sphinx-contrib/youtube && python setup.py install \
-    && wget -q https://raw.githubusercontent.com/ruiztulio/gist-vauxoo/master/travis_run.py \
-    && python travis_run.py \
```

Comment:
I've completed the analysis of [this](https://raw.githubusercontent.com/ruiztulio/gist-vauxoo/master/travis_run.py) script. It does the following things:
1) It clones [odoo-mexico](https://github.com/odoo-mexico/odoo-mexico) on the 7.0 branch.
2) It executes the [install](https://github.com/odoo-mexico/odoo-mexico/blob/7.0/.travis.yml#L13) and [script](https://github.com/odoo-mexico/odoo-mexico/blob/7.0/.travis.yml#L19) sections of the .travis.yml file.
3) The install section clones [maintainer-quality-tools](https://github.com/OCA/maintainer-quality-tools) then executes [travis_install_nightly](https://github.com/oca/maintainer-quality-tools/blob/master/travis/travis_install_nightly).
4) travis_install_nightly exits [here](https://github.com/oca/maintainer-quality-tools/blob/master/travis/travis_install_nightly#L14) because apparently $LINT_CHECK != "0".
5) [travis_install_mx_nightly](https://github.com/odoo-mexico/odoo-mexico/blob/7.0/travis/travis_install_mx_nightly) is executed.
6) Then, on the script section, it clones [addons-vauxoo](https://github.com/vauxoo/addons-vauxoo) and executes [travis_run_tests](https://github.com/OCA/maintainer-quality-tools/blob/master/travis/travis_run_tests).

So, as far as i can tell, the only important steps here are the commands executed by travis_install_nightly in **4** and the commands executed by travis_install_mx_nightly in **5**, which all install lint and odoo dependencies. I'll be adding these dependencies to the script.

What do you think @nhomar, @moylop260, @ruiztulio?